src/OneSignal.ts

```diff
@@ -28,7 +28,7 @@ import SubscriptionModalHost from './modules/frames/SubscriptionModalHost';
 import SubscriptionPopup from './modules/frames/SubscriptionPopup';
 import SubscriptionPopupHost from './modules/frames/SubscriptionPopupHost';
 import OneSignalApi from './OneSignalApi';
-import Popover from './popover/Popover';
```

Comment:
There is still a typo in the commit message :( Popever -> Popover

Reply:
waiting until end so I can rebase and force-push